### PR TITLE
chore: update and extend a bit for 2021

### PR DESCRIPTION
**Pull request by henryiii**
Found a few issues and missing packages.

### CODE REVIEW

**Comment by henryiii** [comment]
Hmm, this seems to be trying to revert #13 - probably bad timing.

File: README.md [hunk 1]
```diff
@@ -40,36 +40,37 @@ Links to other lists of packages are in **bold**.
 ### Reproducibility, reinterpretation
 
    * [Rivet](https://rivet.hepforge.org)
-   * [RECAST](https://recast-docs.web.cern.ch) - [talk](https://indico.cern.ch/event/639314/contributions/2726367/attachments/1542484/2419628/ReinterpretationWorkshopFNAL.pdf)
+   * [RECAST](http://recast.perimeterinstitute.ca) - [talk](https://indico.cern.ch/event/639314/contributions/2726367/attachments/1542484/2419628/ReinterpretationWorkshopFNAL.pdf)
```
Comment (henryiii) [inline README.md]:
```suggestion
   * [RECAST](https://recast-docs.web.cern.ch) - [talk](https://indico.cern.ch/event/639314/contributions/2726367/attachments/1542484/2419628/ReinterpretationWorkshopFNAL.pdf)
```

File: README.md [hunk 2]
```diff
@@ -40,36 +40,37 @@ Links to other lists of packages are in **bold**.
...
+   * [rootpy](http://www.rootpy.org/): _Pythonic_ interface to PyROOT, deprecated.
 
 ### Pure physics
 
-   * [particle](https://github.com/scikit-hep/particle): particle data, such as masses and PDG properties
-   * [decaylanguage](https://github.com/scikit-hep/decaylanguage)
-   * [MadMiner](https://github.com/diana-hep/madminer): Likelihood fits to high-dimensional observables (multi-differential distributions) using Machine-learning techniques.
+   * [particle](https://github.com/scikit-hep/particle): Easy access to particle data, such as masses and PDG properties.
+   * [decaylanguage](https://github.com/scikit-hep/decaylanguage): Read and process or check `.dec` files and also convert AmpGen syntax to GooFit (extendable).
+   * [MadMiner](https://github.com/johannbrehmer/madminer): Likelihood fits to high-dimensional observables (multi-differential distributions) using Machine-learning techniques.
```
Comment (henryiii) [inline README.md]:
```suggestion
   * [MadMiner](https://github.com/diana-hep/madminer): Likelihood fits to high-dimensional observables (multi-differential distributions) using Machine-learning techniques.
```

**Comment by henryiii** [comment]
This also happened to address the things in #12 by moving to the public URLs for uproot/awkward, and noting the deprecation of root-numpy and root-pandas.

**Comment by jpivarski** [comment]
@henryiii Maybe I should have merged this and #12 in the opposite order.

I've resolved the conflicts. Let me know if I've done so as you want it (that I didn't undo any of your edits accidentally), and if you give me the thumbs-up, I'll (squash and) merge it as-is.

File: README.md [hunk 3]
```diff
@@ -93,7 +91,7 @@ Links to other lists of packages are in **bold**.
 
 ### Data processing
 
-   * [awkward-array](https://awkward-array.org): Support "Jagged" arrays (like TTree's) easily in Python. Also supports Numba.
+   * [Awkward Array](https://github.com/scikit-hep/awkward-1.0): Manipulate JSON-like data with NumPy-like idioms.
```
Comment (henryiii) [inline README.md]: Was this intentional? I think `awkward-array.org` is a better (more stable) thing to link to?

File: README.md [hunk 4]
```diff
@@ -78,33 +79,42 @@ Links to other lists of packages are in **bold**.
...
 ### Event generation
 
    * [numpythia](https://github.com/scikit-hep/numpythia)
-   * [pyjet](https://github.com/scikit-hep/pyjet)
+   * [fastjet](https://github.com/scikit-hep/fastjet) (replaces [pyjet](https://github.com/scikit-hep/pyjet))
 
 ### Data processing
 
-   * [awkward-array](https://github.com/scikit-hep/awkward-1.0)
+   * [Awkward Array](https://github.com/scikit-hep/awkward-1.0): Manipulate JSON-like data with NumPy-like idioms.
```
Comment (henryiii) [inline README.md]:
```suggestion
   * [Awkward Array](https://awkward-array.org): Manipulate JSON-like data with NumPy-like idioms.
```

File: README.md (hunk 3 above)
Comment (jpivarski) [reply README.md]: Adding the new catchphrase was intentional, reverting to the repo was not. In the conflict resolution, I don't think I was presented with `https://awkward-array.org` as an option, but you're right that it's a better choice than the GitHub README.

**Pull request merged**